README.md

```diff
@@ -45,11 +45,13 @@ sensor:
         # direction: 900000100002 # Optional stop_id to limit departures for a specific direction (same URL as to find the stop_id)
         # walking_time: 5 # Optional parameter with value in minutes that hides transport closer than N minutes
         # show_official_line_colors: true # Optionally enable official VBB line colors. By default predefined colors will be used.
+        # duration: 30 # Optional (default 10), query departures for how many minutes from now?
```

Comment:
Now we have both “duration” and “walking_time” which basically do the same thing here. Let’s combine them together into one “walking_time” item and use its value as duration parameter in the API

Reply:
Walking_time hides departures closer than N minutes, but I'd like to show the next departures in the next 5-30 minutes, I don't think that would be possible with just one parameter

Reply:
Ah, okay. Now I get it. Allright then